gh-badges/lib/index.spec.js

```diff
@@ -16,5 +21,37 @@ describe('BadgeFactory class', function() {
         template: 'flat',
       })
     ).to.satisfy(isSvg)
+    expect(
+      bf.create({
+        text: ['build', 'passed'],
+        foo: 'bar',
```

Comment:
I was confused by this one at first, but the intent is just a check that we don't want to throw a ValidationError when the user supplies extraneous keys right?

Reply:
exactly

Reply:
I've added  a comment to make this clearer